### PR TITLE
Score and slogans

### PR DESCRIPTION
**Pull request by ChielBruin**
Added scores and slogans!

### CODE REVIEW

File: src/main/java/com/github/fishio/view/SplashScreenController.java [hunk 1]
```diff
@@ -26,12 +27,19 @@
 	private VBox company;
 	@FXML
 	private VBox game;
+	@FXML
+	private Label slogan;
+	
+	private String[] slogans = {"Not even squidding!", "Krilling it!"
+		, "this game is brill-iant!", "In cod we trust!"};
```
Comment (jessearens) [inline src/main/java/com/github/fishio/view/SplashScreenController.java:34]: You are missing a capitol letter in one of your slogans!

**Comment by jessearens** [comment]
I think it looks neat. You only made a grammatical mistake in one of your slogans.

File: src/main/java/com/github/fishio/view/SplashScreenController.java (hunk 1 above)
Comment (Diocruel) [inline src/main/java/com/github/fishio/view/SplashScreenController.java:34]: This means you should fixs this and open a new pull request

Comment (Diocruel) [inline src/main/java/com/github/fishio/view/SplashScreenController.java:34]: I made a grammar mistake too :D

**Pull request closed without merging**